src/components/tooltip/tooltip.spec.js

```diff
@@ -215,10 +216,10 @@ describe('Tooltip', () => {
             it('applies the correct default &:before styles', () => {
               assertStyleMatch(
                 {
-                  borderTop: '7px solid transparent',
+                  borderTop: `${pointerSize}px solid transparent`,
```

Comment:
These tests seem to almost repeat as they assert that the style is applied `before` the element based on the position. You could use `describe.each` for these as well, and could then assert that the style is applied to the border like so 
`border${position}Color: '#C7384F' `

Reply:
I agree, but refactoring of these tests was not in the scope of the task, so I have opted to just update them to test new functionalities.

Reply:
Yeah sorry I've read the ticket and agree it's out of scope